cpp/tests/utilities/column_utilities.cu

```diff
@@ -69,5 +144,56 @@ void expect_equal_buffers(void const* lhs, void const* rhs,
                             typed_rhs));
 }
 
+struct column_view_printer {
+  template <typename Element, typename std::enable_if_t<is_numeric<Element>()>* = nullptr>
```

Comment:
This should work for any fixed-width type (timestamp is fixedwidth, but not numeric). 
```suggestion
  template <typename Element, typename std::enable_if_t<is_fixed_width<Element>()>* = nullptr>
```

Reply:
timestamp fields don't implement operator << or the to_string method.  How would you like them formatted, I can implement to_string for them.

Reply:
This function can't support timestamps.  I added timestamp support below.